code/game/objects/structures/grille.dm

```diff
@@ -10,208 +10,215 @@
-	returnToPool(src)
+	health -= rand(15, 30) //Grille will always be blasted, but chances of leaving things over
+	healthcheck(hitsound = 1)
 
+/obj/structure/grille/meteorhit()
+	health -= rand(15, 30)
+	healthcheck(hitsound = 1)
 
 /obj/structure/grille/Bumped(atom/user)
-	if(ismob(user)) shock(user, 70)
-
+	if(ismob(user))
+		shock(user, 60) //Give the user the benifit of the doubt
 
 /obj/structure/grille/attack_paw(mob/user as mob)
 	attack_hand(user)
 
 /obj/structure/grille/attack_hand(mob/user as mob)
+	var/humanverb = list("kick", "slam", "elbow") //Only verbs with a third person "s", thank you
 	user.delayNextAttack(8)
-	playsound(loc, 'sound/effects/grillehit.ogg', 80, 1)
-	user.visible_message("<span class='warning'>[user] kicks [src].</span>", \
-						 "<span class='warning'>You kick [src].</span>", \
-						 "You hear twisting metal.")
-
-	if(shock(user, 70))
-		return
+	user.visible_message("<span class='warning'>[user] [pick(humanverb)]s \the [src].</span>", \
+	"<span class='warning'>You [pick(humanverb)] \the [src].</span>", \
+	"<span class='warning'>You hear twisting metal.</span>")
 	if(M_HULK in user.mutations)
-		health -= 5
+		health -= 5 //Fair hit
 	else
-		health -= 3
-	healthcheck()
+		health -= 1 //Do minimal damage to encourage people to use literally ANYTHING else
+	healthcheck(hitsound = 1)
+	shock(user, 100) //If there's power running in the grille, allow the attack but grill the user
 
 /obj/structure/grille/attack_alien(mob/user as mob)
-	if(istype(user, /mob/living/carbon/alien/larva))	return
+	if(istype(user, /mob/living/carbon/alien/larva))
+		return
+	var/alienverb = list("slash", "slam", "rip", "claw") //See above
 	user.delayNextAttack(8)
-	playsound(loc, 'sound/effects/grillehit.ogg', 80, 1)
-	user.visible_message("<span class='warning'>[user] mangles [src].</span>", \
-						 "<span class='warning'>You mangle [src].</span>", \
+	user.visible_message("<span class='warning'>[user] [pick(alienverb)]s \the [src].</span>", \
+						 "<span class='warning'>You [pick(alienverb)] \the [src].</span>", \
 						 "You hear twisting metal.")
-
-	if(!shock(user, 70))
-		health -= 5
-		healthcheck()
-		return
+	health -= 5
+	healthcheck(hitsound = 1)
+	shock(user, 75) //Ditto above
 
 /obj/structure/grille/attack_slime(mob/user as mob)
-	if(!istype(user, /mob/living/carbon/slime/adult))	return
+	if(!istype(user, /mob/living/carbon/slime/adult))
+		return
 	user.delayNextAttack(8)
-	playsound(loc, 'sound/effects/grillehit.ogg', 80, 1)
-	user.visible_message("<span class='warning'>[user] smashes against [src].</span>", \
-						 "<span class='warning'>You smash against [src].</span>", \
+	user.visible_message("<span class='warning'>[user] smashes against \the [src].</span>", \
+						 "<span class='warning'>You smash against \the [src].</span>", \
 						 "You hear twisting metal.")
-
-	health -= rand(2,3)
-	healthcheck()
+	health -= 3
+	healthcheck(hitsound = 1)
+	shock(user, 100)

```

Comment:
It makes the grille play shock animations, which is nice. And last I checked the grilles in slime pens were electrified for that purpose, so that's odd